addon/components/power-select-typeahead/trigger.js

```diff
@@ -45,48 +57,67 @@ export default Component.extend({
       } else {
         run.schedule('actions', null, newSelect.actions.open);
       }
-    } else if (!isBlank(newSelect.lastSearchedText) && newSelect.options.length === 0 && newSelect.loading) {
-      run.schedule('actions', null, newSelect.actions.close, null, true);
-    } else if (oldSelect.loading && !newSelect.loading && newSelect.options.length > 0) {
-      run.schedule('actions', null, newSelect.actions.open);
     }
   },
 
-  // Actions
   actions: {
+    /**
+     * on mousedown prevent propagation of event
+     *
+     * @private
+     * @method stopPropagation
+     * @param {Object} event
+     */
     stopPropagation(e) {
       e.stopPropagation();
     },
 
+    /**
+     * called from power-select internals
+     *
+     * @private
+     * @method handleKeydown
+     * @param {Object} event
+     */
     handleKeydown(e) {
+      // up or down arrow and if not open, no-op and prevent parent handlers from being notified
       if ([38, 40].indexOf(e.keyCode) > -1 && !this.get('select.isOpen')) {
         e.stopPropagation();
         return;
       }
       let isLetter = e.keyCode >= 48 && e.keyCode <= 90 || e.keyCode === 32; // Keys 0-9, a-z or SPACE
+      // if isLetter, escape or enter, prevent parent handlers from being notified
       if (isLetter || [13, 27].indexOf(e.keyCode) > -1) {
+        let select = this.get('select');
+        if (!select.isOpen) {
+          run.schedule('actions', null, select.actions.open);
```

Comment:
@cibernox this should only open right away to show `loadingMessage` if loadingMessage is configured.  However, at this level, I am not sure how to tell if a loadingMessage was configured or not.  I have failing tests below.  

Any thoughts?